### PR TITLE
bots: Post learn-tests training data only to centosci

### PR DESCRIPTION
**Pull request by Gundersanne**

### CODE REVIEW

File: bots/learn-trigger [hunk 1]
```diff
@@ -45,7 +45,7 @@ def main():
         "tests-data " + TRAINING_DATA,
         # We want this to run in both namespaces
         "learn-tests http://cockpit-learn.cockpit.svc.cluster.local/train/" + TRAINING_DATA,
```
Comment (martinpitt) [inline bots/learn-trigger]: Is that URL still a thing then?

Comment (Gundersanne) [reply bots/learn-trigger]: so this should be the url for the centosci clutser, this won't work if the task starts on bos. However we'd need to expose the webdav part of the server which is a pain. But the alternative is always hoping the task would run on centosci which is also a pain.

**Comment by Gundersanne** [comment]
@martinpitt  I've run this task (learn-tests) on a task bot locally and it worked fine. I'd like try it out on the infra.

File: bots/learn-tests [hunk 2]
```diff
@@ -98,25 +98,19 @@ def train(filename, url, verbose=False):
...
     # Passing through a non terminal stdout is necessary to make progress work
-    curl = subprocess.Popen(cmd, stdout=subprocess.PIPE)
-    cat = subprocess.Popen(["cat"], stdin=curl.stdout)
-    curl.stdout.close()
-    ret = curl.wait()
-    cat.wait()
-    if ret != 0:
+    try:
+        subprocess.check_call(cmd)
+    except:
```
Comment (martinpitt) [inline bots/learn-tests]: No bare excepts please. `subprocess.CalledProcessError` seems fine. But why bother with that and re-raising a different exception? IMHO you can just let it fail, the original command line already has the url.

Comment (Gundersanne) [reply bots/learn-tests]: woops, sorry about that. changed :)

**Pull request merged**